### ISSUE TITLE
accessibility - focus styles

### ISSUE THREAD

**Issue by twinlensreflex** [mxstbr/sharing#75]
there are no focus styles in the css! this is an accessibility issue. Ideally for accessibility you would have an equivalent `:focus` for any `:hover` style.

**Comment by Hassan-Ib** [comment]
can I look into this ?.
